src/app/containers/Blocks/__snapshots__/index.test.jsx.snap

```diff
@@ -1,200 +1,15 @@
-              Object {
-                "id": null,
-                "model": Object {
-                  "attributes": Array [],
-                  "text": "This is some text content!",
-                },
-                "type": "fragment",
-              },
-            ],
-            "text": "This is some text content!",
-          },
-          "type": "paragraph",
-        },
-      ]
-    }
-    type="text"
-    typeOfPreviousBlock="subheadline"
-  />,
-]
+<p>
+  Some text
+</p>
 `;
 
 exports[`Blocks with mixed set of known and unknown blocks should render known blocks; unknown blocks rendered null 1`] = `
-Array [
-  <headline
-    blocks={
-      Array [
-        Object {
-          "id": null,
-          "model": Object {
-            "blocks": Array [
-              Object {
-                "id": null,
-                "model": Object {
-                  "blocks": Array [
-                    Object {
-                      "id": null,
-                      "model": Object {
-                        "attributes": Array [],
-                        "text": "This is a headline!",
-                      },
-                      "type": "fragment",
-                    },
-                  ],
-                  "text": "This is a headline!",
-                },
-                "type": "paragraph",
-              },
-            ],
-          },
-          "type": "text",
-        },
-      ]
-    }
-    type="headline"
-  />,
-  <subheadline
-    blocks={
-      Array [
-        Object {
-          "id": null,
-          "model": Object {
-            "blocks": Array [
-              Object {
-                "id": null,
-                "model": Object {
-                  "blocks": Array [
-                    Object {
-                      "id": null,
-                      "model": Object {
-                        "attributes": Array [],
-                        "text": "This is a subheadline!",
-                      },
-                      "type": "fragment",
-                    },
-                  ],
-                  "text": "This is a subheadline!",
-                },
-                "type": "paragraph",
-              },
-            ],
-          },
-          "type": "text",
-        },
-      ]
-    }
-    type="subheadline"
-    typeOfPreviousBlock="headline"
-  />,
-  <text
-    blocks={
-      Array [
-        Object {
-          "id": null,
-          "model": Object {
-            "blocks": Array [
-              Object {
-                "id": null,
-                "model": Object {
-                  "attributes": Array [],
-                  "text": "This is some text content!",
-                },
-                "type": "fragment",
-              },
-            ],
-            "text": "This is some text content!",
-          },
-          "type": "paragraph",
-        },
-      ]
-    }
-    type="text"
-    typeOfPreviousBlock="subheadline"
-  />,
-  null,
-  null,
-]
+<p>
```

Comment:
Lots of stuff missing here, any ideas why Its not rendering stuff like `This is a headline!`?